### PR TITLE
new submitfilter

### PR DESCRIPTION
**Pull request by stdweird**

### CODE REVIEW

**Comment by stdweird** [comment]
probably needs more unittests etc etc

**Comment by boegel** [comment]
@stdweird: my guess is that you'll need to assign someone to review this beast...

File: bin/submitfilter.py [hunk 1]
```diff
@@ -33,261 +34,113 @@
...
-
-MIN_VMEM = 1536 << 20  # minimum amount of ram in our machines.
-
-
-def main(arguments=sys.argv):
+def make_new_header(sf):
     """
-    main method
+    Generate a new header by rewriting selected options and add missing ones.
     """
```
Comment (itkovian) [inline bin/submitfilter.py]: What is 'sf'?

**Comment by stdweird** [comment]
@itkovian im closing this PR, it's unmergebale with master. new one will appear soon

**Pull request closed without merging**